include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -48,84 +48,77 @@ namespace __internal
 //-----------------------------------------------------------------------------
 
 #if _ONEDPL_DEBUG_SYCL
-template <typename _ExecutionPolicy>
-::std::string
-__device_info(const _ExecutionPolicy& __policy)
+inline std::string
+__device_info(const sycl::queue& __q)
 {
-    return __policy.queue().get_device().template get_info<sycl::info::device::name>();
+    return __q.get_device().template get_info<sycl::info::device::name>();
```

Comment:
Since we no longer have the `_ExecutionPolicy` template, I think the scenario is no longer ambiguous and `template` can be removed here.